lib/configure-robot.js

```diff
@@ -93,17 +94,17 @@ module.exports = async (app) => {
       delayMs: 0, // disable delaying - full speed until the max limit is reached
     });
 
-    app.router.use(limiter);
+    router.use(limiter);
   }
 
   // These incoming webhooks should skip rate limiting
   setupGitHub(app);
   // According to NewRelic, we receive < 1 request per minute from Jira
-  setupJira(app);
+  setupJira(app, { getRouter });
   // Our FrontEnd Assets are between 1 and 4 RPM
-  setupFrontend(app);
+  setupFrontend(app, { getRouter });
   // Opaque is expected to query us twice per minute
-  setupPing(app);
+  setupPing(app, { getRouter });
```

Comment:
just fyi this works for Probot v12, but in a future release we will remove the `{ getRouter }` option in order to make probot app function universal across traditional node servers and serverless environments. I recommend you create your own server using express and then use the [`createNodeMiddleware` export from `probot`](https://probot.github.io/docs/development/#use-createnodemiddleware) only for the route where you receive the webhooks.